src/project-tests/survey-form-tests.js

```diff
@@ -115,9 +115,17 @@ export default function createSurveyFormTests() {
             });
 
             it('12. I can select an option from a dropdown that has corresponding id="dropdown".', function() {
-                var dropdown = document.getElementById('dropdown');
-                FCC_Global.assert.isNotNull(dropdown, 'There should be a select field with id="dropdown" ');
-                FCC_Global.assert.isAtLeast(getNumOptions(dropdown.options), 2, "Select should contain at least 2 selectable options ");
+                var dropDown = document.getElementById('dropdown');
```

Comment:
@tbushman sorry for being picky, but let's change this to `drowdown` since this is just one word. that also means `dropDownTag` should just be `dropdownTag`. 

Thanks!